dashboard/components/DevDashboard.vue

```diff
@@ -60,15 +62,26 @@ const campaignList = computed( (): Campaign[] => Object.values( props.campaigns
 const currentCampaign = computed( (): Campaign => campaignList.value.find( ( c: Campaign ) => c.campaign === branchName.value ) );
 const filteredCampaignList = computed( (): Campaign[] => campaignList.value.filter( ( c: Campaign ) => c.campaign !== branchName.value ) );
 
-onMounted( () => {
-	fetch( '/compiled-banners/' )
-		.then( async res => {
-			if ( !res.ok ) {
-				return;
-			}
-			const fileList = await res.text();
-			compileInfo.value = parseCompileInfo( fileList );
+const campaignUrlMap = ref<Record<string, string>>( {} );
+
+onMounted( async () => {
+	try {
+		const bannersResponse = await fetch( '/compiled-banners/' );
+		if ( !bannersResponse.ok ) {
+			return;
+		}
+		const fileList = await bannersResponse.text();
+		compileInfo.value = parseCompileInfo( fileList );
```

Comment:
While we can expect the `/compiled-banners/` route to work reliably (it's provided by the webpack dev server), I want to point out that the `return` on error in lines 70-72 will prevent the querying of the GitHub API if an error occurs.
Also, the `await` prevents the browser loading *both* URLs at the same time. 
This is no blocker and I'll still approve the code. What you could do instead, to make the two fetch requests independent from each other and parallel would be to move the two code snippets to their own function (that function doesn't need try/catch) and then use `Promise.all` to wait for their completion like this:

```typescript
try {
  await Promise.all(fetchCompileInfo(), fetchPullRequestUrlsFromGitHub());
} catch ( error ) {
  console.error( 'Error:', error.message );
}
```

Reply:
Done.